docs/_data/sidebar-fx.yml

```
@@ -37,18 +40,25 @@ options:
       - title: Semaphore
         url: /apidocs/arrow-fx-coroutines/arrow.fx.coroutines/-semaphore/
 
-  - title: Integrations
+  - title: Coroutines Integrations
+
+    nested_options:
+
+      - title: kotlinx.coroutines
+        url: /coroutines-integrations/kotlinxcoroutines/
```

```suggestion
        url: /integrations/kotlinxcoroutines/
```

I think that's the permalink added in the document.